### PR TITLE
Add crecto back

### PR DESCRIPTION
**Pull request by wontruefree**
Crecto is back!
https://github.com/Crecto/crecto/issues/233#issuecomment-531911736
https://github.com/Crecto/crecto/commit/9c86cd89dafe493ee8e015169f9dff66b4bb7ba8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +219,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [tarantool-crystal](https://github.com/vladfaust/tarantool-crystal) - Tarantool driver
 
 ## Database Tools
+ * [crecto-admin](https://github.com/Crecto/crecto-admin) - Admin dashboard for Crecto and your database
```
Comment (hugoabonizio) [inline README.md]: Is Crecto Admin still working with latest Crystal changes?

Comment (wontruefree) [reply README.md]: It seems like some updates have happened recently but i don't think crecto has had an official release since support.

Comment (hugoabonizio) [reply README.md]: I'm glad to see that Crecto's back! However, maybe we should ask for @fridgerator's input about Crecto Admin?

**Comment by fridgerator** [comment]
Crecto is working, however I havent tested crecto admin yet.  I probably can sometime this week.

**Comment by veelenga** [comment]
@fridgerator just tried crecto admin, and it doesn't work.

@wontruefree  Please remove it from this PR to unblock the  other addition.

**Comment by fridgerator** [comment]
Crecto admin relies on kemal-session, which needs to be updated.
https://github.com/kemalcr/kemal-csrf/issues/22

After this, it should work.

If this PR is blocking something, then feel free to remove it.

**Comment by wontruefree** [comment]
@veelenga I think both can be added back.  Let me know what you think

**Comment by veelenga** [comment]
@wontruefree i would be glad to, but i still not able to even build crecto-admin

image

**Pull request merged**